features/dd-sdk-android-trace/api/dd-sdk-android-trace.api

```diff
@@ -413,85 +424,501 @@ public final class com/datadog/exec/DaemonThreadFactory : java/util/concurrent/T
 	public fun newThread (Ljava/lang/Runnable;)Ljava/lang/Thread;
 }
 
-public class com/datadog/opentracing/DDSpan : com/datadog/trace/api/interceptor/MutableSpan, io/opentracing/Span {
```

Comment:
are we going to verify binary compatibility of Java classes exposed through our Kotlin API? Do we break anything?

Reply:
these classes are exposed only for internal usage, we do not have a Java sample app for now but I did some try - outs. In any case these classes should not be used by our customers directly. It is the same case for the old Java code.

Reply:
Well, they were still publicly available (since Java doesn't have `internal` visibility modifier) and customer could downcast `Span` to `DDSpan` to do something, we didn't make any statement in the Javadoc saying they shouldn't be considered as a part of the public API. But I guess at the current point there is nothing we can do.